projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -94,18 +94,31 @@ module.exports = async function (...args) {
 		buildSoy();
 	}
 
-	runBabel(
-		BUILD_CONFIG.input,
-		'--out-dir',
-		BUILD_CONFIG.output,
-		'--source-maps'
-	);
+	const disableOldBuild =
+		FEDERATION_CONFIG && FEDERATION_CONFIG.disableOldBuild;
+
+	if (!disableOldBuild) {
+		runBabel(
+			BUILD_CONFIG.input,
+			'--out-dir',
+			BUILD_CONFIG.output,
+			'--source-maps'
+		);
+	}
 
-	if (fs.existsSync('webpack.config.js') || !!FEDERATION_ENABLED) {
+	if (fs.existsSync('webpack.config.js') || !!FEDERATION_CONFIG) {
```

Comment:
nit: `!!` is unneeded

Reply:
Oh, yes. We added it [here](https://github.com/liferay/liferay-frontend-projects/pull/252#pullrequestreview-539930843) but it was a different context. :+1: